ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java

```diff
@@ -276,39 +276,86 @@ public void getCompletedJobJson() throws Exception {
+            Util.copyFile(objectName, Util.RESOURCE_BASE_NAME, Util.DOWNLOAD_BASE_NAME);
+            File newFile = new File(Util.DOWNLOAD_BASE_NAME + "/" + objectName);
+            DateTime now = new DateTime();
+            newFile.setLastModified(now.getMillis());
 
+            Util.createBucket(client, bucketName);
+            Util.loadBookTestData(client, bucketName);
+
+            final Arguments args = new Arguments(new String[]{"--http", "-c", "get_object", "-b", bucketName,
+                    "-o", objectName, "-d", Util.DOWNLOAD_BASE_NAME, "--sync"});
+            CommandResponse response = Util.command(client, args);
+            assertThat(response.getMessage(), is("SUCCESS: No need to sync " + objectName));
         } finally {
             Util.deleteBucket(client, bucketName);
-            Util.deleteLocalFile("beowulf.txt");
+            Util.deleteLocalFile(objectName);
         }
     }
 
     @Test
     public void getBulkObjectWithSync() throws Exception {
-        final String bucketName = "test_get_bulk_object_with_sync";
+        final String bucketName = "test_get_bulk_object_with_sync_no_update";
         try {
+            Util.createBucket(client, bucketName);
+            Util.loadBookTestData(client, bucketName);
+
+            final Arguments args = new Arguments(new String[]{"--http", "-c", "get_bulk", "-b", bucketName,
+                    "-d", Util.DOWNLOAD_BASE_NAME, "--sync"});
+            CommandResponse response = Util.command(client, args);
+            assertThat(response.getMessage(), is("SUCCESS: Synced all the objects from " + bucketName + " to ." + File.separator + "." + File.separator + "output"));
+        } finally {
+            Util.deleteBucket(client, bucketName);
+            Util.deleteLocalFiles();
+        }
+    }
 
+    @Test
+    public void getBulkObjectWithSyncNoUpdate() throws Exception {
+        final String bucketName = "test_get_bulk_object_with_sync_no_update";
+        try {
             Util.createBucket(client, bucketName);
             Util.loadBookTestData(client, bucketName);
 
             final Arguments args = new Arguments(new String[]{"--http", "-c", "get_bulk", "-b", bucketName,
                     "-d", Util.DOWNLOAD_BASE_NAME, "--sync"});
             CommandResponse response = Util.command(client, args);
-            assertThat(response.getMessage(), is("SUCCESS: Synced all the objects from test_get_object to ." + File.separator + "." + File.separator + "output"));
+            assertThat(response.getMessage(), is("SUCCESS: Synced all the objects from " + bucketName + " to ." + File.separator + "." + File.separator + "output"));
+
+            // get_bulk sets the last_modified property to that of the original file, so we need to spoof for comparison
+            final File destFolder = new File(Util.DOWNLOAD_BASE_NAME);
+            DateTime now = new DateTime();
```

Comment:
I thought i set this to be an error in IntelliJ, must be per project.  thanks, will fix